components/molecules/CardWithImageLayout.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types'
+
+export default function CardWithImageLayout(props) {
+  return (
+    <div className={'flex flex-col sm:flex-row w-full ' + props.layoutClasses}>
+      <div className={props.cardClasses + ' flex '}>{props.card}</div>
```

Comment:
I don't know if we need these div's. Seems a little redundant to have div's on top of div's.

An option would be to change this:
![image](https://user-images.githubusercontent.com/77116011/136599645-6225c0d6-ef5f-4f92-afd8-81db08bc9a5c.png)
 
 
 To something like this:
 
![image](https://user-images.githubusercontent.com/77116011/136599683-ee4128e2-24e2-4d74-a13e-1a5e41a4a6a9.png)